examples/org-viewer-react/src/OrgGroups.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core'
+import { Fragment } from 'react'
+import { useOrganization } from '@aragon/connect-react'
+import OrgApps from './OrgApps'
+import OrgInfo from './OrgInfo'
+import OrgPermissions from './OrgPermissions'
+
+type OrgGroupsProps = { onOpenApp: (appAddress: string) => void }
+
+export default function OrgGroups({ onOpenApp }: OrgGroupsProps) {
+  const [org, { loading }] = useOrganization()
+  return (
```

Comment:
It's not a big deal here, but do we have an explicit contract that `org` will be unavailable until `loading` is false?

Or can you have a partial `org` object that is continually streamed in?

Reply:
At the moment, `org` is `null` until it finishes loading, but it only does it once. Ideally I think we should keep streaming continually yes (that would require a new method on the connector).